.github/dependabot.yml

```diff
@@ -1,11 +1,14 @@
 version: 2
 updates:
   - package-ecosystem: 'npm'
-    versioning-strategy: lockfile-only
+    versioning-strategy: increase-if-necessary
```

Comment:
I believe this will allow dependabot to do major version bumps as well. Do we want this?

Reply:
In the commit message:

>This does introduce the possibility of breaking changes, as increase-if-necessary will update to a updating a major version. However since all PRs require a review, we can catch these breaking changes before they become a problem. We can monitor this and adjust the strategy if necessary.